docs/api/users-create-user.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+---
+
+# Create user
+
+Creates a [`user`](user) who is registering with the to-do service. The request body contains the new user details. You must specify the required properties for the user. 
```

Comment:
```suggestion
Creates a [`user`](user) who is registering with the to-do service.
The request body contains the new user details.
You must specify the required properties for the user. 
```

If you start each sentence on a new line, it renders the same but it's easier to spot future edits to any one of the sentences.